kyuubi-events/src/main/scala/org/apache/kyuubi/events/EventBus.scala

```diff
@@ -42,6 +45,19 @@ sealed trait EventBus {
 
 object EventBus extends Logging {
   private val defaultEventBus = EventBusLive()
+  private val conf: KyuubiConf = KyuubiConf()
+  conf.loadFileDefaults()
+
+  private val poolSize = conf.get(ASYNC_EVENT_HANDLER_POLL_SIZE)
+  private val waitQueueSize = conf.get(ASYNC_EVENT_HANDLER_WAIT_QUEUE_SIZE)
+  private val keepAliveMs = conf.get(ASYNC_EVENT_HANDLER_KEEPALIVE_TIME)
+
+  private val syncEventExecutionContext =
+    ExecutionContext.fromExecutorService(ThreadUtils.newDaemonQueuedThreadPool(
```

Comment:
The global `ExecutionContext` is `ForkJoinPool`, we are using `ThreadPoolExecutor` here, which one is suitable for us to use here? Or is there another better one?

Reply:
I don't have any suggestions here, because I haven't used asynchronous mode in the actual scenario. If I want to handle the event with asynchronous mode,  prefer to implement it in the `EventHandler`. we can control the event cache in the EventHandler to reduce the number of threads.

Reply:
However, it is user-friendly to provide the default asynchronous mode.